testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/IdentityProviderTest.java

```diff
@@ -646,6 +652,10 @@ public void testSamlImportAndExportMultipleSigningKeys() throws URISyntaxExcepti
         form.addFormData("file", body, MediaType.APPLICATION_XML_TYPE, "saml-idp-metadata-two-signing-certs");
 
         Map<String, String> result = realm.identityProviders().importFrom(form);
+        //firtsly check and remove enabled from result
+        boolean enabled = Boolean.valueOf(result.get("enabled"));
+        result.remove("enabled");
```

Comment:
Use constant, not literal: 
```suggestion
        boolean enabled = Boolean.valueOf(result.get(ENABLED_FROM_METADATA));
        result.remove(ENABLED_FROM_METADATA);
```